src/pyseekdb/cli.py

```
@@ -0,0 +1,311 @@
+def collection_info(ctx, name, peek_count):
+    """Show detailed information about a collection.
+
+    NAME is the collection name to inspect.
+    """
+    console = Console()
+
+    try:
+        with get_client(ctx) as client:
+            try:
+                coll = client.get_collection(name)
+            except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)